velox/dwio/parquet/tests/ParquetReaderTest.cpp

```diff
@@ -87,8 +87,12 @@ class ParquetReaderTest : public testing::Test {
     while (total < expected->size()) {
       auto part = reader.next(1000, result);
       EXPECT_GT(part, 0);
-      assertEqualVectorPart(expected, result, total);
-      total += part;
+      if (part > 0) {
+        assertEqualVectorPart(expected, result, total);
+        total += part;
+      } else {
+        break;
```

Comment:
`EXPECT_GT(part, 0)` will always ensure `part > 0` correct? When is the need for the `if-else`?

Reply:
EXPECT_GT, unlike ASSERT_GT, is not preventing the following line from existing. Hence, the check is needed to prevent crashes.